README.md

```diff
@@ -33,22 +33,14 @@ To setup your workspace run the following commands at the root of the project
 
 1.1). Setup nvm
 
-```
-> brew install nvm
-> vim ~/.zshrc
-  # add the follow lines to your .zshrc file
-  export NVM_DIR="$HOME/.nvm"
-  source "$NVM_DIR/nvm.sh"
-> esc
-> :wq
-```
+Follow the install directions at https://github.com/nvm-sh/nvm#install--update-script
 
-***Make sure to use new terminals once modified `~/.zshrc`***
+***Make sure to use new terminals after completing install***
 
 
 ```
-> nvm install v14.19.0
-> nvm use v14.19.0
+> nvm install v16.14.0
```

Comment:
Thanks! Can you mention updating the readme as a step in `NODE_UPGRADES.md` too?